dev-lang/v8/v8-7.0.276.28.recipe

```diff
@@ -0,0 +1,139 @@
+srcGitRev8="d5266326752f0a1dadbd310932d8f4fd8c3c5e7d"
+SOURCE_URI_8="git+https://chromium.googlesource.com/external/github.com/google/googletest.git#$srcGitRev8"
+
+PATCHES="v8-$portVersion.patch"
+PATCHES_3="build-$portVersion.patch"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	SECONDARY_ARCHITECTURES="x86"
+	ARCH=x86
+else
+	ARCH=x64
+fi
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+    commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	v8$secondaryArchSuffix = $portVersion
+	cmd:d8$commandSuffix = $portVersion
+	lib:libv8$secondaryArchSuffix = $portVersion
+	lib:libv8_for_testing$secondaryArchSuffix = $portVersion
+	lib:libv8_libbase$secondaryArchSuffix = $portVersion
+	lib:libv8_libplatform$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libexecinfo$secondaryArchSuffix
+	lib:libssp$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	v8${secondaryArchSuffix}_devel = $portVersion
+	devel:libv8$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	v8$secondaryArchSuffix == $portVersion
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	gcc${secondaryArchSuffix}_syslibs
+	gcc${secondaryArchSuffix}_syslibs_devel
+	llvm7$secondaryArchSuffix
+	llvm7${secondaryArchSuffix}_libs
+	devel:libgcc$secondaryArchSuffix
+	devel:libexecinfo$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:clang
+	cmd:clang++
+	cmd:gcc$secondaryArchSuffix
+	cmd:g++$secondaryArchSuffix
+	cmd:gn$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:ninja
+	cmd:python
+	cmd:which
+	"
+
+BUILD()
+{
+	cd $sourceDir
+
+	mkdir -p base/trace_event/common
+	mkdir -p build
+	mkdir -p buildtools
+	mkdir -p third_party/jinja2
+	mkdir -p third_party/markupsafe
+	mkdir -p tools/clang
+	mkdir -p third_party/googletest/src
+
+	cp -rf /sources-2/* $sourceDir/base/trace_event/common
+	cp -rf /sources-3/* $sourceDir/build/
+	cp -rf /sources-4/* $sourceDir/buildtools
+	cp -rf /sources-5/* $sourceDir/third_party/jinja2
+	cp -rf /sources-6/* $sourceDir/third_party/markupsafe
+	cp -rf /sources-7/* $sourceDir/tools/clang
+	cp -rf /sources-8/* $sourceDir/third_party/googletest/src
+
+	mkdir -p out.gn
+
+	gn gen out.gn/$ARCH.release --args="target_os=\"haiku\" target_cpu=\"$ARCH\"
+        v8_enable_embedded_builtins = false is_debug = false symbol_level=0
+        is_clang=true treat_warnings_as_errors=false use_sysroot=false
+        v8_enable_i18n_support=false is_component_build=true"
+
+	ninja -C out.gn/$ARCH.release $jobArgs
+}
+
+INSTALL()
+{
+	mkdir $commandBinDir
+	cp -af out.gn/$ARCH.release/d8 ${commandBinDir}
+	mkdir -p ${includeDir}/v8
+	cp -af include/*  ${includeDir}/v8
+	mkdir -p $libDir
+	cp -af out.gn/$ARCH.release/*.so ${libDir}
+
+	prepareInstalledDevelLibs libv8
+	packageEntries devel $developDir
+}
+
+TEST()
+{
+	echo "Test"
+	# Disable this for now, KDLs on some tests
```

Comment:
Please report the KDLs and include reduced methods to reproduce them if possible. But at least report them.

Reply:
I've re-enabled tests and omitted the ones that cause KDLs. I'll see if I can find a way to reproduce this KDL.